client/stylesheets/_toCat.scss

```diff
@@ -2094,6 +2110,13 @@ i:hover {
 .helpThree li ul li p {
     margin: 0 0 2px 0;
 }
+
+.helpThree li ul li:last-child {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `helpThree` should be written in lowercase with hyphens